src/platform/vscode-path/resources.ts

```diff
@@ -8,12 +8,12 @@ import { Uri as URI } from 'vscode';
 import { CharCode } from './charCode';
 import * as extpath from './extpath';
 import * as paths from './path';
-import { isLinux, isWindows } from './platform';
+import { isLinux, isWeb, isWindows } from './platform';
 import { compare as strCompare, equalsIgnoreCase } from './strings';
 import { Schemas } from './utils';
 
-function originalFSPath(uri: URI): string {
-    return uri.fsPath;
+export function originalFSPath(uri: URI | undefined): string {
```

Comment:
I might not have payed enough attention to where this was used, but the name seems a bit off now. A .path property for a web URI isn't an fspath.

Reply:
Actually looking at the usage I think this works. NVMD